src/test/java/software/amazon/awssdk/crt/test/CrtTestFixture.java

```diff
@@ -33,6 +35,11 @@ public final CrtTestContext getContext() {
 
     @Before
     public void setup() {
+        if (System.getProperty("aws.crt.aws_trace_log_per_test") != null) {
```

Comment:
Isn't this the same code as the fixture?

Reply:
yep, we have two fixtures. One for the S3 native client test, the other one is for the regular CRT test